README.md

```diff
@@ -10,7 +29,52 @@ curl -H 'Cache-Control: no-cache' https://raw.githubusercontent.com/fossas/spect
 Set-ExecutionPolicy Bypass -Scope Process -Force; iex  ((New-Object System.Net.WebClient).DownloadString('https://raw.githubusercontent.com/fossas/spectrometer/master/install.ps1'))
 ```
 
-## Building
+**NOTE**: In order to use the executable, you may need to add it to your PATH.
+The installer will report the installation path of the executable to make this
+easier.
+
+## Basic Usage
+
+The tool requires little-to-no configuration to run on its own.
+
+```sh
+fossa2 analyze --output # Analyze the current working directory
+# OR
+fossa2 analyze --output --basedir /path/to/project  # Analyze a specific directory.
+```
+
+The `--output` flag tells the analyzer not to connect to a FOSSA server,
+instead printing out the analysis result to the console.
+
+While we're on the subject, you can (and should!) connect your output directly
+to your FOSSA server!  You can do this by simply supplying the FOSSA API key
+from your server ([more info](https://docs.fossa.com/docs/api-reference)) and
+using it as shown:
+
+```sh
+export FOSSA_API_KEY=your-key-goes-here
+fossa2 analyze
+# OR
+fossa2 analyze --fossa-api-key your-key-goes-here
+```
+
+This not only lets you re-examine the results later, but allows you to check
+your output against your preset policies.
+
+**NOTE** If leaked, your FOSSA API key can grant an attacker access to your FOSSA
+projects, and should be kept secret.  For this reason, we reccommend supplying
+the API key with the environment variable, especially in a shared environment
+like a CI/CD server.
+
+## Supported Languages/Managers
```

Comment:
I'd just call this "Supported Languages".

Reply:
That's a bit of a misnomer, considering that mostly we're analyzing files related to package managers, not languages.  I don't like the current title, but I'd like to revisit this.

Reply:
I agree that it's a bit of a misnomer, but I would prefer easier comprehension over more pedantic correctness in user documentation.

Reply:
"Pedantic" might be a little strong.  But I don't disagree with the rest of your point.  I'll change it to "Supported Package Managers".